setup.yml

```diff
@@ -13,6 +13,7 @@
     - flannel
     - master
     - addons
+    - dnsmasq
```

Comment:
does the sequencing of when to install and configure dnsmasq need to be last or maybe just a simple dependency to the role kubernetes since both master and minion require it?